### ISSUE TITLE
Quick questions...

### ISSUE THREAD

**Issue by bonelifer** [p-doyle/Python-ReadBluetoothScale#1]
Two questions. First, is the weight just output as just the number? Second, this command only needs to be run once?

`sudo gatttool -b <scale_mac>  --char-write --handle=0x0013 --value=0x1309150210b91d0019`

Nice work. Been looking online for something inexpensive that still allowed me to get the data on computer automatically without having to send it to the cloud. Everything I found was for some one off machine that hadn't been sold since 2008, 2013, 2016 or if you could find it on eBay, they were selling them used for what they probably brought back then. The closest I found was using the wii Balance board. It looks great, but would end up costing two as much, because you'd need a 3rd party AC adapter that cost as much as you can find the boards for and would need some sort of wedge to push the reset/sync button to allow it to connect, because permanent pairing(from non-wii systems) seems a bit flaky from the comments I've read. Thanks, just hit the buy button on eBay.

**Comment by p-doyle** [comment]
The data from the scale is returned as bytes but that gets converted to an integer in the script.  And I believe so, though I am not sure if it would persist if you changed it to lbs and pulled the battery.  And actually that command only changes what gets displayed on the scale itself I think, lbs or kgs.  The data sent via bluetooth is always in kg. 

Lol I went down the same road...  you'd think someone would have made a scale that was easier to interface with...

FYI I've made a few changes since creating the repo that I just pushed.  The original method I had been using had some issues when the script was run over longer periods of time.  The downside is that to get it to work reliably I had to set the cooldown to 15 minutes so if you need to take measurements more frequently than that you'll need to make some adjustments.

**Comment by bonelifer** [comment]
So, I'm a below the knee amputee on my right leg. While waiting on my USB Bluetooth dongle for my laptop, I tested out the scale with the phone app, while standing with my walker. I noticed it was showing the weight on the app, but not allowing me to save it automatically. Etekcity support confirmed what I feared, unless the scale can read all the other metrics, the app won't save the weight. Is your script relying on the scale to tell you it's done, or are you using your own logic?  Also, I tried using the command above with my mac address but it just sat there doing nothing till I CTRL-C'ed out of it.

**Comment by p-doyle** [comment]
Yeah, it seems it uses the metal pads on the scale to detect skin contact I think to determine if there is actually a person on the scale or something...  The script doesn't rely on that though and will pull the weight as soon as it starts blinking on the LCD, I didn't even bother looking for the register it was using to check for that since it didn't seem necessary.

I am not sure if this is the issue but for gatttool to be able to connect the scale has to be awake so you'll have to put some weight on it to wake it up before using the command.
